_data/roadmap.yml

```diff
@@ -78,32 +78,28 @@
     - name: 0MQ/ZeroMQ
       date: September, 2017
       status: completed
-    - name: Fluffy blocks
```

Comment:
Why remove this? Would be better if marked as completed imho.

Reply:
It was a duplicate entry, Fluffyblocks were already marked as completed above.

https://github.com/mattcode55/monero-site/blob/30300c95661dea9fec172337863e861e1423692a/_data/roadmap.yml#L69-L70

Reply:
Oh okay, thanks!